CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java

```diff
@@ -214,7 +215,7 @@ public String getPropertyFromEnv(String key, String defaultValue){
 	}
 	
 	public ArrayList<String> getRelatedHosts(String envId) {
-		String[] relates = getProperty("env." + envId + ".ssh.host.related", "").split(",");
+		String[] relates = getProperty(ConfigParameterConstants.TEST_INSTANCE_PREFIX  + envId + ConfigParameterConstants.TEST_INSTANCE_HA_SLAVE_SUFFIX, "").split(",");
```

Comment:
Rename TEST_INSTANCE_HA_SLAVE_SUFFIX to TEST_INSTANCE_RELATED_HOSTS_SUFFIX.

And there is a missing dot.